tests/acceptance/pageObjects/FilesPageElement/publicLinksDialog.js

```diff
@@ -333,11 +333,9 @@ module.exports = {
       return this.waitForElementVisible(copyBtnSelector).click(copyBtnSelector)
     },
     copyPrivateLink: function() {
-      const appSideBarElements = this.api.page.FilesPageElement.appSideBar().elements
-      const sidebarLinksTab = appSideBarElements.sidebarLinksTab.selector
-      const sidebarCss = appSideBarElements.sideBar.selector
-
-      return this.waitForElementVisible(sidebarCss)
+      const sidebarLinksTab = this.api.page.FilesPageElement.appSideBar().elements
```

Comment:
ultra nit pick: here is one more `tab` in a var name left

Reply:
fix